tft_bot/league_api/league_api_integration.py

```diff
@@ -145,7 +109,7 @@ def connect_to_lcu(self, wait_for_availability: bool = False) -> bool:
                 "Accept": "application/json",
             }
         )
-        self._session.verify = f"{ROOT_CERTIFICATE_PATH}\\riotgames_root_certificate.pem"
+        self._session.verify = system_helpers.resource_path("tft_bot/resources/riotgames_root_certificate.pem")
```

Comment:
Seems a bit odd to me that we set the verification cert here & in L332 -- is this necessary? 🤔

Reply:
It's two separate session objects. We can probably re-use the same session object since the settings are the same, but I separated them just in case we need to have differing configurations in the future 🤷🏻 I don't mind merging them into one util method though, if you prefer :)

Reply:
It's two separate session objects. We can probably re-use the same session object since the settings are the same, but I separated them just in case we need to have differing configurations in the future 🤷🏻 I don't mind merging them into one util method though, if you prefer :)

Reply:
Ah gotcha, yeah that's fine enough for me -- if we notice issues we can refactor then! 😛